cms/djangoapps/contentstore/views/component.py

```diff
@@ -116,7 +116,7 @@ def container_handler(request, usage_key_string):
 
             # Fetch the XBlock info for use by the container page. Note that it includes information
             # about the block's ancestors and siblings for use by the Unit Outline.
-            xblock_info = create_xblock_info(xblock, include_ancestor_info=is_unit_page)
+            xblock_info = create_xblock_info(xblock, include_ancestor_info=is_unit_page, escape_html=False)
```

Comment:
@agaylard Can you explain why you think this parameter is needed?  I would assume that this name is never escaped until it is used in a template, at which point it would be escaped as needed using the proper filters based on context.